src/hook/pauseResume.js

```diff
@@ -0,0 +1,318 @@
+/**
+* Copyright (c) 2015 NAVER Corp.
+* egjs projects are licensed under the MIT license
+*/
+eg.module("pauseResume", ["jQuery"], function($) {
+	"use strict";
+
+	var animateFn = $.fn.animate;
+	var stopFn = $.fn.stop;
+	var uuid = 1;
+	var getStyles;
+
+	if (window.getComputedStyle) {
```

Comment:
@mixed 
I considered to use jQuery method and I found that `getStyle` function is implemented as a local function. So I looked for external function that wraps the getStyle. But I couldn't find.

I found 2 use cases of `getStyle` on external function. 
  1. $.cssHooks["height" | "width"].set = function() {}
  2. $.fn.css

First one is not correspond with purpose, 
Second one needs property name to get specific value.

Can you help me to find the substitution for the above code? :)

And I can't understand the situation where dash style is included. Do you mean that `getComputedStyle` can return dash-style value? or element can have dash-style value (So it cannot be obtained through `getComputedStyle`)